presidio-analyzer/presidio_analyzer/analyzer_engine.py

```diff
@@ -176,3 +179,18 @@ def __remove_low_scores(
 
         new_results = [result for result in results if result.score >= score_threshold]
         return new_results
+
+    def __remove_analysis_explanation(self, results: List[RecognizerResult],
```

Comment:
Is it possible to not generate the interpretability info instead of removing it?

Reply:
Internally, the traces use it.
So we want to keep it inside, but not always to return in via the HTTP payload.